app/src/parser.c

```diff
@@ -335,34 +326,39 @@ __Z_INLINE parser_error_t parser_screenPrint(const parser_context_t *ctx,
         return parser_ok;
     }
 
-    // Complete screen
-    char key[MAX_KEY_LENGTH + 2] = {0};
-    MEMCPY(key, tmp_key, MAX_KEY_LENGTH);
-    for (uint8_t i = 0; i < container->screen.indent; i++) {
-        z_str3join(key, sizeof(key), SCREEN_INDENT, "");
-    }
-
-    if (strlen(val) == 0) {
-        return parser_unexpected_value;
-    }
-
-    val += (val[0] == ' ' ? 1 :0);
-    if (val[strlen(val)-1] == ' ' || val[strlen(val)-1] == '@' ) {
-        if (strlen(val)+1 > OUTPUT_HANDLER_SIZE) {
-            return parser_unexpected_value;
+    //Translate output, cpy to tmp to assure it ends in \0
+    MEMZERO(ctx->tx_obj->tx_text.tmpBuffer, sizeof(ctx->tx_obj->tx_text.tmpBuffer));
```

Comment:
nit: you might define tmpLen = sizeof(ctx->tx_obj->tx_text.tmpBuffer) and then use the auxiliary variables instead of this long ctx->tx_obj..... to refer to the temporal buffer ;)